portfolio/src/main/webapp/style.css

```diff
@@ -109,12 +109,19 @@ a:hover {color:#560d32;}
   flex-wrap: nowrap;
   flex: 50%;
   max-width: 50%;
-  padding: 4px;
+  padding-left: 4px;
+  padding-right: 4px;
+  margin-left: 4px;
```

Comment:
Do you mind alphabetizing these? I think the "margin-left" and "margin-right" should come before the max-width :)